volume/struct.go

```diff
@@ -26,6 +26,10 @@ const (
 	VolStopped
 )
 
+var (
+	MValidateBrickPathStats = utils.ValidateBrickPathStats
```

Comment:
Does MValidateBrickPathsStats need to be exposed?  Also, just curious, but why start with MV?